### DISCUSSION TITLE
流水线节点(Agent) - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1142] [Announcements]
# 流水线节点(Agent) - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/pipeline/config/agent/

**Comment by muma378** [comment]
agent还包含二进制：helm（2.11.0），helm3（3.5.0），kubectl（latest，和具体发版的时间有关）

**Comment by muma378** [comment]
看下 上面的格式问题呢

> bash apiVersion: v1 kind: Pod spec: containers: - name: maven image: maven:3.8.1-jdk-8 command: - sleep args: - 99d - name: golang image: golang:1.16.5 command: - sleep args: - 99d
